Makefile

```diff
@@ -542,7 +595,7 @@ prepare-e2e-kuttl-tests: build docker build-assert-job
 kuttl-e2e: prepare-e2e-kuttl-tests start-kind run-kuttl-e2e
 
 .PHONY: run-kuttl-e2e
-run-kuttl-e2e:
+run-kuttl-e2e: kafka
```

Comment:
I think this should be  part of test prepare target.

Reply:
Sure! I fixed this for #1598 but I didn't have the chance to apply the change here